README.md

```diff
@@ -275,6 +295,12 @@ cd ~/Downloads/data
 ls -al
 ```
 
+## Changelogs
+
+### 1.0.1
```

Comment:
There's a change in the error handling, too -- please mention it.

Reply:
Done.